contracts/duckies/nft/DucklingFamilyGame/DucklingFamilyGameV1.sol

```diff
@@ -0,0 +1,245 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.18;
+
+import '@openzeppelin/contracts-upgradeable/utils/CountersUpgradeable.sol';
+import '@openzeppelin/contracts-upgradeable/utils/StringsUpgradeable.sol';
+import '@openzeppelin/contracts-upgradeable/utils/cryptography/ECDSAUpgradeable.sol';
+
+import '../../../interfaces/IVoucher.sol';
+import '../RandomUpgradeable.sol';
+import '../Gene.sol';
+
+contract DucklingFamily is IVoucher, RandomUpgradeable {
+	using CountersUpgradeable for CountersUpgradeable.Counter;
+	using Gene for uint256;
```

Comment:
Rename to Genome

Reply:
71402bebaffa9e16a8bee156a31acafbd13d9bff